mautic/mautic
# Social login

## Description
Added new field options when creating a form for a Facebook login button and a g+ login button
buttons auto fill matched fields on form with social profile data as long as form field names are the same as social data fields
when form is submitted it creates or updates leads with social profile data

## Steps to test
In Setting Manage Plugins
-Enable social network plugin
-Client credentials must be entered in the social plugin and authorized
-Select Login button in the configuration features
-Use Contact Field Mapping to map the social network's data to those on the contact

On the form:
-Create a new form
-Add a new button on form using the Add Field option Social Login
-Once selected, the social networks enabled in the settings should show
-Use the social buttons to fill in the fields that would match social data fields this will also create or edit contacts. You should see: fields filled from the Contact Field Mapping filled in the contact located, and the social cache updated.
